features/matchers/have_broadcasted_matcher.feature

```diff
@@ -0,0 +1,151 @@
+Feature: have_broadcasted matcher
```

Comment:
Don't we need documentation for other matchers too? Documentation display only one matcher doc. https://relishapp.com/palkan/action-cable-testing/docs/matchers/have-broadcasted-matcher

Reply:
Yeah, I think it makes sense to add a _matchers_ doc. We already have some example group specific matchers there (like `render_template`), so it should be OK.

Update the gem's docs: https://relishapp.com/palkan/action-cable-testing/docs/matchers/have-stream-from-matcher

If it looks good I update the PR as well.

Reply:
LGTM. Thanks